src/components/NavBar/NavMenu.jsx

```diff
@@ -55,13 +56,28 @@ const NavMenu = ({
       sx={{ mt: 5, backgroundColor: 'rgba(1, 121, 105, 0.2)' }}
     >
       <MenuList>
+        {/* messages */}
+        <MenuItem
+          component={Button}
+          startIcon={<EmailIcon />}
+          sx={{ display: { md: 'none' }, color: theme.palette.primary.main, width: '100%' }}
+        >
+          <NavLink
+            to="/PASS/Messages"
```

Comment:
Looks like this wasn't updated yet to the new path name. It's causing the mobile version to have a runaway useEffect.

https://github.com/codeforpdx/PASS/assets/14917816/61d1cf2b-afae-47f9-8330-c15ebb33328d

It looks like we need to make the DocumentsList responsive too, but we can save that for another PR.

Reply:
Yeah there's a separate issue for the ClientsListTable, so might as well make one for the DocumentsListTable